src/hooks/useGlobalState.tsx

```diff
@@ -1,13 +1,12 @@
 import React, { useContext, useReducer } from 'react'
-import { GlobalState } from 'reducers-actions'
 import { AnyAction } from 'combine-reducers'
 
 const GlobalStateContext = React.createContext({})
 
-export function withGlobalContext<P>(
+export function withGlobalContext<P, State>(
```

Comment:
lol nice, fixing my shitty types

Reply:
Nothing really wrong with your type, thinking that there shouldn't be more than 1 global state.